src/promptflow/promptflow/_core/connection_manager.py

```diff
@@ -55,7 +55,7 @@ def _build_connections(cls, _dict: Dict[str, dict]):
                 secret_keys = connection_dict.get("secret_keys", [])
                 secrets = {k: v for k, v in value.items() if k in secret_keys}
                 configs = {k: v for k, v in value.items() if k not in secrets}
-                connection_value = connection_class(configs=configs, secrets=secrets)
+                connection_value = connection_class(name=key, configs=configs, secrets=secrets)
```

Comment:
@brynn-code and @thy09 to help comment on this `core` change.

Reply:
Sure, without this change connection manager does not return the correct name of the connection. See the highlighted section in the screenshot below.
![image](https://github.com/microsoft/promptflow/assets/75808410/05638290-bd15-475a-ad7b-7f49e5f0990d)

Reply:
This is no longer needed in my tests. I have removed the change.